uniffi_bindgen/src/bindings/swift/templates/macros.swift

```diff
@@ -57,14 +57,15 @@
 // Note the var_name and type_name filters.
 -#}
 {% macro field_list_decl(item) %}
-    {%- for field in item.fields() -%}
+    {% for field in item.fields() -%}
```

Comment:
Minimal nit: I don't think you need to remove the `-` here. Then again it saves us on some whitespace and that's really not critical.

Reply:
Indeed, fixed it.